website/next.config.js

```diff
@@ -31,4 +33,6 @@ module.exports = {
       "/": { page: "/overview/introduction" },
     };
   },
+  assetPrefix: isProd ? `/halstack/${process.env.SITE_VERSION}` : undefined,
```

Comment:
How these env variables are set? I can¡t find the place where we are assigning values to them

Reply:
This environment variable is set by default by NextJS.